sqlite/migrations/0002_create_annotations_tables.sql

```diff
@@ -0,0 +1,36 @@
+-- The user_version should match the "000X" from the file name
+-- Ex: 0001_create_notebooks_table should have a user_verison of 1
+PRAGMA user_version=2;
+
+-- Create the initial table to store streams
+CREATE TABLE streams (
+  id VARCHAR(16) UNIQUE NOT NULL,
+  org_id VARCHAR(16) NOT NULL,
+  name TEXT NOT NULL,
+  description TEXT NOT NULL,
+  created_at TIMESTAMP NOT NULL,
+  updated_at TIMESTAMP NOT NULL,
+
+  PRIMARY KEY ("org_id", "name")
+);
+
+-- Create the initial table to store annotations
+CREATE TABLE annotations (
+  id VARCHAR(16) UNIQUE NOT NULL,
+  org_id VARCHAR(16) NOT NULL,
+  stream_id VARCHAR(16) NOT NULL,
+  name TEXT NOT NULL,
```

Comment:
Should `name` have a uniqueness constraint?

Reply:
Soo....the `name` column is actually supposed to be called `stream`, which refers to the stream name (same one as the `stream_id`). Since multiple annotations can be part of the same stream, it doesn't need to be unique. I updated the DB schema and struct tags/queries to use `stream` instead of `name`. This will actually match the cloud schema.

Reply:
Hmmm could we get away with not tracking the stream name here, and instead get it by joining annotations to streams on the FK at query time? That's how I'd do it in psql but not sure if sqlite has some trade-offs that make denormalization better.